src/order/filter/countable_Inter.lean

```diff
@@ -42,7 +42,7 @@ lemma countable_sInter_mem {S : set (set α)} (hSc : S.countable) :
 ⟨λ hS s hs, mem_of_superset hS (sInter_subset_of_mem hs),
   countable_Inter_filter.countable_sInter_mem' hSc⟩
 
-lemma countable_Inter_mem [encodable ι] {s : ι → set α} :
+lemma countable_Inter_mem {ι : Sort*} [countable ι] {s : ι → set α} :
```

Comment:
You are making iota a Sort* here, but it is still a Type* in most lemmas in this file. Would it be worth it making it a Sort* globally in this file, or does it create problems?